### PR TITLE
Improve description for NotificationMode

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/11361

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ Accept wildcard characters: False
 
 ### -NotificationMode
 The type of conference experience for security desk notification.
+Possible values are NotificationOnly, ConferenceMuted, ConferenceUnMuted.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md:159]:
```suggestion
Possible values are NotificationOnly, ConferenceMuted, and ConferenceUnMuted.
```

File: skype/skype-ps/skype/Set-CsTeamsEmergencyCallingPolicy.md [hunk 2]
```diff
@@ -151,6 +151,7 @@ Accept wildcard characters: False
 
 ### -NotificationMode
 The type of conference experience for security desk notification.
+Possible values are NotificationOnly, ConferenceMuted, ConferenceUnMuted.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsEmergencyCallingPolicy.md:154]:
```suggestion
Possible values are NotificationOnly, ConferenceMuted, and ConferenceUnMuted.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JensTrier — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by msbemba — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**